app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,53 @@
+class OrdersController < ApplicationController
+  include CartHelper
+
+  before_action :logged_in_user, :load_product_in_cart,
+                only: :create
+  before_action :load_order, only: :show
+
+  def index_by_user
+    @orders = Order.load_order_by_user current_user.id
```

Comment:
```suggestion
    @orders = current_user.orders
```
ri mắc cha j mi tạo cái scope thế cu